app/routers/api/storage.py

```
@@ -300,8 +331,9 @@ def delete_storage_location(
             logger.warning(f"Path not found, proceeding with DB deletion: {location.path}")
         except Exception as e:
             logger.exception(
-                f"Error deleting storage directory '{location.path}': {e}. "
-                f"Manual cleanup may be required."
+                f"Error deleting storage directory '{location.path}'. "
+                f"Manual cleanup may be required.",
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)